src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -56,6 +56,7 @@ protected void SetUpEditor(string markupCode)
             {
                 VisualStudio.Editor.SetText(code);
                 VisualStudio.Editor.MoveCaret(caretPosition);
+                VisualStudio.Editor.Activate();
```

Comment:
:memo: This is based on the idea that #17393 is one of the following:

* A focus bug (command is sent to the wrong handler chain)
* A timing bug (command is sent before the command handler is registered)

`Activate` seemed like a low-risk way to rule out the first case.